.travis.yml

```diff
@@ -60,4 +60,5 @@ install:
     - "pip freeze"
 
 script:
+    - flake8 mapproxy
```

Comment:
To check: not sure if it would be desirable to make the build fail early when there are linting errors. If we want that, I just need to check how to do that with travis. I feel the easy way would be `flake8 mapproxy && pytest mapproxy`, but maybe there is a more travis-elegant way.

Reply:
dont rely on travis, it may get replaced soon, see https://github.com/mapproxy/mapproxy/issues/487